src/views/Today.vue

```diff
@@ -4,8 +4,7 @@
       <img class="artworkBg" src="../assets/Oval.svg" />
       <img class="artwork" src="../assets/Artwork.svg" />
     </div>
-    <br />
-    <br />
+    <div class="spacing-m"></div>
```

Comment:
Instead of creating empty div tag with class, add margin top to element below or margin bottom to element above this div.

Reply:
https://github.com/nsftx/incubator-2019-rating-app-admin-client/pull/45/commits/ab655400e583cc4928bfc09a5648f3e55a68e023